### ISSUE TITLE
APBS and compute charge

### ISSUE THREAD

**Issue by huanggewudi** [vsomnath/holoprot#4]
4d7b: Could not compute charges because of Charges cannot be computed. Missing file. 4d7b_out.csv. Trying with fixed file.
4d7b: Could not compute charges for fixed file because of Charges cannot be computed. Missing file. 4d7b_out.csv. Returning None

APBS 1.5 cant work.  I have APBS 3.4.1 but some file cant compute charge.(apbs cant get xxx.out.csv)

**Comment by huanggewudi** [comment]
when I just run APBS, get this
error while loading shared libraries: libamd.so.1: cannot open shared object file: No such file or directory

**Comment by vsomnath** [comment]
Hi, thanks for reaching out!

There are always examples where computing charges fails - I am unable to use them for training / testing. 
If the number of examples where this fails is small, then you could consider ignoring them. 

I am sharing my binaries configuration in the next comment - maybe this helps your cause!

**Comment by huanggewudi** [comment]
Thank you, I have about five percent of the pdbs in your data set that cannot calculate the charge. I found that it is because of using pdb2pqr to report an error on these pdbs. （TypeError: '<' not supported between instances of 'PotentialBond' and 'PotentialBond'）

I want to know that when you conduct experiments, the pdb that cannot calculate the charge is directly discarded. Or give some default initial value？Also wondering if you can't calculate the charge on the data set you gave?

Thanks again！

**Comment by vsomnath** [comment]
I just discard the pdb altogether - as far as I remember, across the test set - it was very few examples that got discarded as a result so I went ahead with this strategy. 

I have added a link to the binaries in the README (under the Environment Variables section) - let me know if that works for you!

Preprocessing is a major limitation and hacky component of HoloProt. I am working on a version that removes this preprocessing altogether, and instead uses a combination of HoloProt and dMaSIF but time constraints on other things ensure that this is slow.
